content/_help/get-started/overview._es.md

```diff
@@ -5,31 +5,11 @@ meta_title: Visión general
 permalink: /es/help/get-started/overview/
 order: 0
 redirect_from:
-  - /es/help/creating-an-account/authentication-application/
-  - /es/help/creating-an-account/creating-a-strong-password/
-  - /es/help/creating-an-account/email-address-confirmation-link-is-invalid/
   - /es/help/creating-an-account/email-address-confirmation-not-received/
-  - /es/help/creating-an-account/email-address-is-already-in-use/
-  - /es/help/creating-an-account/how-to-create-an-account/
-  - /es/help/creating-an-account/no-phone-or-other-authentication-method/
-  - /es/help/creating-an-account/personal-key/
-  - /es/help/creating-an-account/phone-didnt-get-a-security-code/
-  - /es/help/creating-an-account/security-key/
-  - /es/help/creating-an-account/two-factor-authentication/
-  - /es/help/creating-an-account/do-i-need-a-mobile-phone-to-use-logingov/
-  - /es/help/creating-an-account/how-do-i-create-an-account-with-logingov/
-  - /es/help/creating-an-account/how-do-i-create-an-account-with-only-one-two-factor-authenticator/
-  - /es/help/creating-an-account/how-do-i-verify-my-address-by-mail/
-  - /es/help/creating-an-account/how-do-i-verify-my-identity-by-mail/
   - /es/help/creating-an-account/i-didnt-receive-a-confirmation-email-from-logingov/
-  - /es/help/creating-an-account/what-do-i-do-if-an-account-already-exists-under-my-email-address/
-  - /es/help/creating-an-account/what-is-my-personal-key/
-  - /es/help/creating-an-account/what-is-two-factor-authentication/
-  - /es/help/creating-an-account/why-didnt-i-receive-a-security-code-to-confirm-my-phone/
-  - /es/help/creating-an-account/why-do-i-need-to-confirm-my-email-address-and-my-phone-number/
-  - /es/help/creating-an-account/why-do-i-need-to-use-logingov-to-access-government-services-online/
   - /es/help/creating-an-account/why-is-my-confirmation-link-invalid/
   - /es/help/get-started/
+  - /help/creating-an-account/why-do-i-need-to-use-logingov-to-access-government-services-online/
```

Comment:
I wouldn't expect to see this here? It's not Spanish, and it's already defined above in `overview._en.md`.

```suggestion
```

Reply:
That should not have been there. It's now removed.